### PR TITLE
Show the original author on cloned casebooks.

### PR DESCRIPTION
**Pull request by cgruppioni**
* Save casebook ancestry when cloning a casebook with the Ancestry gem 
* Remove export button from section & resources 
* style tweaks to dashboard and edit details (title, etc ) section for a draft casebook
* some naming and haml -> erb refactoring

### CODE REVIEW

File: app/views/content/dashboard/_content_browser.html.erb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+          <div class="owner">
+            <%= casebook.owner.attribution %><span class="<%= casebook.owner.verified? ? 'verified' : '' %>"></span>
+          </div>
+          <div class="affiliation">
+            <%= casebook.owner.affiliation %>
+          </div>
+        </div>
+        <div class="root-attribution">
+          <% if casebook.ancestry %>
+            Original author: <%= casebook.root.root_owner %>
```
Comment (edwardloveall) [inline app/views/content/dashboard/_content_browser.html.erb]: Did you mean for this to be a link?

File: db/migrate/20171030175029_add_ancrestry_to_content_nodes.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class AddAncrestryToContentNodes < ActiveRecord::Migration[5.1]
+  def up 
+    add_column :content_nodes, :ancestry, :string
+    add_index :content_nodes, :ancestry
```
Comment (edwardloveall) [inline db/migrate/20171030175029_add_ancrestry_to_content_nodes.rb]: Keep it for now, but in the future, you can do this in one line:

```ruby
add_column :content_nodes, :ancestry, :string, index: true
```

File: app/views/content/dashboard/_content_browser.html.erb (hunk 1 above)
Comment (cgruppioni) [reply app/views/content/dashboard/_content_browser.html.erb]: Not meant to be a link 👍

File: db/migrate/20171030175029_add_ancrestry_to_content_nodes.rb (hunk 2 above)
Comment (cgruppioni) [reply db/migrate/20171030175029_add_ancrestry_to_content_nodes.rb]: Ah yeah

Comment (bensteinberg) [reply db/migrate/20171030175029_add_ancrestry_to_content_nodes.rb]: Typo: Ancrestry should be Ancestry in file and class name

**Pull request merged**